st2client/st2client/commands/action.py

```diff
@@ -282,31 +282,21 @@ def __init__(self, resource, *args, **kwargs):
         super(ActionDeleteCommand, self).__init__(resource, *args, **kwargs)
 
         self.parser.add_argument(
-            "-f",
-            "--force",
+            "-rf",
```

Comment:
Single hyphen flags should be a single letter only.  Under Linux, it's possible to have flags grouped by a single hyphen which would make the `-rf` flag seem like two flags `-r -f` were being passed. `-r` should be sufficient for the `remove_files` flag unless it's already taken.